src/test/java/nl/atos/ontwikkelstraat/FormControllerSteps.java

```diff
@@ -0,0 +1,146 @@
+package nl.atos.ontwikkelstraat;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import nl.atos.ontwikkelstraat.pojo.FormData;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+public class FormControllerSteps {
+    private final WebDriver webdriver;
+    public FormData formData;
+
+    public FormControllerSteps(SharedDriver webdriver) {
+        this.webdriver = webdriver;
+    }
+
+    @Before
+    public void setUp() {
+        formData = new FormData("john", "banks", "frontstreet", 12, "9485KS");
+    }
+
+    @After
+    public void tearDown() {
+        formData = null;
+    }
+
+    @Given("^That the website is running and I navigated to it$")
+    public void that_the_website_is_running_and_I_navigated_to_it() throws InterruptedException {
+        webdriver.manage().window().maximize();
+        webdriver.manage().timeouts().setScriptTimeout(10, TimeUnit.SECONDS);
+        if (System.getenv("TRAVIS") == "true") {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2345692422/issues/source?bid=3138095&fileBranchId=3208141#l41)